src/middleware/builtin.spec.ts

```diff
@@ -162,6 +162,30 @@ describe('matchMessage()', () => {
     );
     it('should filter out message events which do not have text (block kit)', noTextMessageTestCase(pattern));
   });
+
+  describe('using an array pattern', () => {
+    const pattern = ['foobar', /foo/];
+    const matchingText = 'foobar';
+    const nonMatchingText = 'bar';
```

Comment:
In the case of a different pattern array, you'd want foo to match, bar to match, foobar to match and then another example that's non matching.

Reply:
Fixed, in accordance with this suggestion.